components/async-status-mgt/org.wso2.carbon.identity.framework.async.status.mgt/src/test/java/org/wso2/carbon/identity/framework/async/status/mgt/constants/TestAsyncOperationConstants.java

```diff
@@ -0,0 +1,59 @@
+package org.wso2.carbon.identity.framework.async.status.mgt.constants;
+
+/**
+ * Constants related to the test cases of Async Operation Status Management component.
+ */
+public class TestAsyncOperationConstants {
+
+    //Organizations.
+    public static final String RESIDENT_ORG_ID_1 = "10084a8d-113f-4211-a0d5-efe36b082211";
+    public static final String RESIDENT_ORG_ID_2 = "c524c30a-cbd4-4169-ac9d-1ee3edf1bf16";
+    public static final String RESIDENT_ORG_ID_3 = "cd5a1dcb-fff2-4c14-a073-c07b3caf1757";
+    public static final String RESIDENT_ORG_ID_4 = "7cb4ab7e-9a25-44bd-a9e0-cf4e07d804dc";
+    public static final String RESIDENT_ORG_ID_5 = "440ad5b6-6a41-4da7-aadd-272995d0e5db";
+
+    // Tenant Domains.
+    public static final String TENANT_DOMAIN_1 = "d6fc0b22-5b7f-461b-aef8-775a78187095";
+
+    // Correlation ID.
+    public static final String CORR_ID_1 = "440abcd-6a41-4da7-aadd-272995d0e5db";
+    public static final String CORR_ID_1_PREFIX = "440abcd-6a41-4da7-aadd-272995d0e5d";
+    public static final String CORR_ID_2 = "550abcd-6a41-4da7-aadd-272995d0e5db";
+    public static final String CORR_ID_3 = "660abcd-6a41-4da7-aadd-272995d0e5db";
+    public static final String CORR_ID_4 = "770abcd-6a41-4da7-aadd-272995d0e5db";
+
+    //Operation Subjects.
+    public static final String SUBJECT_ID_1 = "448e57e7-ff6b-4c31-a1eb-2a0e2d635b2a";
+    public static final String SUBJECT_ID_1_PREFIX = "448e57e7-ff6b-4c31-a1eb-2a0e2d635b2";
+    public static final String SUBJECT_ID_2 = "558e57e7-ff6b-4c31-a1eb-2a0e2d635b2a";
+    public static final String SUBJECT_ID_3 = "668e57e7-ff6b-4c31-a1eb-2a0e2d635b2b";
+    public static final String SUBJECT_ID_4 = "778e57e7-ff6b-4c31-a1eb-2a0e2d635b2c";
+    public static final String SUBJECT_ID_5 = "888e57e7-ff6b-4c31-a1eb-2a0e2d635b2c";
+
+    //Operation Initiators.
+    public static final String INITIATOR_ID_1 = "443k57a1-gl6b-4c31-a1eb-2a0e2d635b2a";
+    public static final String INITIATOR_ID_2 = "553k57a1-gl6b-4c31-a1eb-2a0e2d635b2a";
+    public static final String INITIATOR_ID_3 = "663k57a1-gl6b-4c31-a1eb-2a0e2d635b2b";
+    public static final String INITIATOR_ID_4 = "773k57a1-gl6b-4c31-a1eb-2a0e2d635b2c";
+    public static final String INITIATOR_ID_5 = "883k57a1-gl6b-4c31-a1eb-2a0e2d635b2c";
+
+    //Operation Types.
+    public static final String TYPE_USER_SHARE = "B2B_USER_SHARE";
+    public static final String TYPE_USER_BULK_IMPORT = "USER_BULK_IMPORT";
+    public static final String TYPE_APP_SHARE = "B2B_APPLICATION_SHARE";
+
+    //Operation Subject Types.
+    public static final String SUBJECT_TYPE_USER = "USER";
+    public static final String SUBJECT_TYPE_APPLICATION = "APPLICATION";
+
+    //Operation Status.
+    public static final String STATUS_SUCCESS = "SUCCESS";
+    public static final String STATUS_FAIL = "FAIL";
+    public static final String STATUS_PARTIAL = "PARTIAL";
```

Comment:
The term PARTIAL is ambiguous.
It does not clearly indicate whether the process has partially succeeded and completed, or if it is still in progress and partially processed.